variables.tf

```diff
@@ -94,6 +94,24 @@ variable "arn_format" {
   default     = "aws"
 }
 
+variable "create_dynamodb_table" {
+  type        = bool
+  description = "Boolean for if a DynamoDB table is to be created for batch locking."
+  default     = true
+}
+
+variable "user_managed_dynamodb_table_name" {
+  type        = string
+  description = "Name of the user-managed DynamoDB table."
+  default     = null
+}
+
+variable "dynamodb_table_ttl" {
```

Comment:
can we skip this since it's already set in the Python and it's a little confusing since the infra doesn't actually use it?